main.ledger

```diff
@@ -1545,12 +1580,26 @@
     Liabilities:Reimbursement:Zach Latta
     ; Receipt: a2875e2a6987e90e4817b121274cfdf6.pdf
      
+2016/02/03 Someone didn't cash their check
+    Assets:Bank:Checking                              $1900.00
+    Expenses:Salary
+    ; Receipt: 9b1b2f9c1dcc2d6a19757f88fa422fcb.png
```

Comment:
you should have gotten some sort of email from wells fargo; we should use that as the receipt instead